app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -67,13 +67,15 @@ def stop_cockpit_ws
   def stop_cockpit_ws_process
     return unless @pid
     Process.kill("TERM", @pid)
+    @stdout&.close
+    @stderr&.close
```

Comment:
Should these be closed before the Process is TERM'd?

Reply:
> Should these be closed before the Process is TERM'd?

make sense, most likely no any output needed just before killing process.